### PR TITLE
disabling buttons a bit after usage

### PR DESCRIPTION
**Pull request by someaddons**
disables buttons temporarily after usage

### CODE REVIEW

File: src/main/java/com/minecolonies/blockout/controls/Button.java [hunk 1]
```diff
@@ -6,6 +6,10 @@
 import net.minecraft.init.SoundEvents;
 import org.jetbrains.annotations.NotNull;
 
+import javax.swing.Timer;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/blockout/controls/Button.java:11]: Java awt stuff is quite often not well seen in these projects, isn't there a java.util Timer as well?

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/3246/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Review by marchermans — COMMENTED** [review]
Why?

**Comment by Raycoms** [comment]
Since we got a good bunch of issues with people spam clicking buttons
before the action has been completed.

Am Mi., 9. Jan. 2019 um 18:26 Uhr schrieb Marc Hermans <
notifications@github.com>:

**Comment by marchermans** [comment]
I do not think this is the way to go. Most buttons people are having problems with, can be disabled straight up. Timers just cause threading problems......

**Comment by Raycoms** [comment]
The best way to go might be just using the render ticks to unblock them.
It's ticking there anyway

Am Mi., 9. Jan. 2019 um 20:38 Uhr schrieb Marc Hermans <
notifications@github.com>:

**Comment by marchermans** [comment]
@Raycoms that would be a perfect solution

**Pull request closed without merging**